File: neetcode.io/01-arrays/02-Anagram/go/main.go
// Copyright (c) 2023 Furukawa, Atsushi <atsushifx@aglabo.com>
//
// This software is released under the MIT License.
// https://opensource.org/licenses/MIT

package main

// modules
import (
	"fmt"
)

// main
func main() {
	var ans bool
	var s, t string

	s = "anagram"
	t = "nagaram"
	ans = isAnagram(s, t)
	fmt.Printf("[%s , %s] %v\n", s, t, ans)

	s = "rat"
	t = "car"
	ans = isAnagram(s, t)
	fmt.Printf("[%s , %s] %v\n", s, t, ans)
}

File: neetcode.io/01-arrays/02-Anagram/typescript/main.ts
// Copyright (c) 2023 Furukawa, Atsushi <atsushifx@aglabo.com>
//
// This software is released under the MIT License.
// https://opensource.org/licenses/MIT

// import solver
import solver from './solver.js';

main()
// main
function main() {
	let ans: boolean;	
	let s: string
	let t: string

	s = "anagram";
	t = "nagaram"
	ans = solver.isAnagram(s, t)
	console.log(s + ", " + t + " answer : " + ans)

	s = "rat";
	t = "car"
	ans = solver.isAnagram(s, t)
	console.log(s + ", " + t + " answer : " + ans)
}

